### DISCUSSION TITLE
cli debug command fails

### DISCUSSION THREAD

**Discussion by stefaanMLB** [mikro-orm/mikro-orm#4267] [Q&A]
in a new nestjs project, I get the below error message when using the micro-orm `debug command`
```
$ npx mikro-orm debug
Current MikroORM CLI configuration
 - dependencies:
   - mikro-orm 5.7.0
   - node 18.13.0
TypeError: Cannot read properties of undefined (reading 'length')
    at C:\Users\lcartreul\Documents\projecten\staalophaling-api\node_modules\@mikro-orm\cli\cli.js:14:16
```
I have this setup in `package.json`
```json
  "mikro-orm": {
    "debug": true,
    "tsConfigPath": "./tsconfig.json",
    "useTsNode": true,
    "configPaths": [
      "./src/mikro-orm.config.ts"
    ]
  }
```

ts-node is installed
I tried variations with `mikro-orm.js` file instead, same result
Any ideas what's going wrong ?

**Comment by B4nan** [comment]
This is already fixed in master (#4265), you can use 5.7.1-dev.2+ version.

**Comment by stefaanMLB** [comment]
OK, I downgraded to 5.6.16
Thanks for the help !
